sceptre/scipool/config/strides/bootstrap.yaml

```diff
@@ -0,0 +1,4 @@
+template:
```

Comment:
I'm not sure how this file got removed.  The stack does exist in the account so I am restoring the file.

Reply:
figured it out from PR #995 bootstrap.yaml should be removed from this location

Reply:
fixed